pkg/config/odfi.go

```diff
@@ -245,6 +245,10 @@ type Storage struct {
 	// after downloading and processing of each file.
 	KeepRemoteFiles bool
 
+	// RemoveZeroByteFilesAfter is the amount of time we want to wait before deleting
+	// zero byte files.
```

Comment:
```suggestion
	// The amount of time (in minutes) to wait before deleting zero byte files.
```

Reply:
godoc typically wants the name as the first word